### PR TITLE
container-logs-panel: Use a monospaced font

### PR DESCRIPTION
**Pull request by SoMuchForSubtlety**
Also hides the cursor, since the TextView is not for input.

Please let me know if that's not the right place or method to set those properties, I'm very new to rust and GTK.

### CODE REVIEW

**Comment by SoMuchForSubtlety** [comment]
I just saw that you already added this in [logs-search](https://github.com/marhkb/pods/tree/logs-search)

**Comment by marhkb** [comment]
That's ok. I'd like to encourage collaboration, so I'd like to merge it so it's already in the main branch. The other PR can still take a little while.
If you don't mind I would review it then.

File: src/view/container/container_logs_panel.rs [hunk 1]
```diff
@@ -96,7 +98,8 @@ mod imp {
 
         fn constructed(&self, obj: &Self::Type) {
             self.parent_constructed(obj);
-
+            self.text_view.set_property("monospace", true);
+            self.text_view.set_property("cursor-visible", false);
```
Comment (marhkb) [inline src/view/container/container_logs_panel.rs:102]: You could set these properties directly in the .ui file

**Review by marhkb — CHANGES_REQUESTED** [review]

**Comment by marhkb** [comment]
Thank you!

**Pull request merged**